refactor: replace deprecated pkg_resources API and drop Python 3.8 support (#45)
eduNEXT/xblock-filesmanager: CHANGELOG.rst

Unreleased
**********

1.0.0 - 2025-02-05
**********************************************

Changed
=======

> * Remove python 3.8 and django 3.2 support

Reviewer: ```suggestion
* Remove support for Python 3.8 and Django 3.2.
```

> * pkg_resources is deprecated, and its recommended replacement, importlib.resources, is only fully available in Python 3.9+.

Reviewer: Same comment as https://github.com/eduNEXT/eox-theming/pull/81#discussion_r1948413825

Unreleased
**********

1.0.0 - 2025-02-05
**********************************************

Changed
=======

> * Remove support for Python 3.8 and Django 3.2.

Reviewer: ```suggestion
* Removed support for Python 3.8 and Django 3.2.
```

* Replaced deprecated `pkg_resources` APIs with `importlib.resources` and `importlib.metadata` to ensure compatibility with newer Django and Python versions.

> * Starting with this version (1.0.0), it is no longer compatible with Quince.

Reviewer: ```suggestion
* Starting with this version, this package is no longer compatible with releases older than Redwood.
```
